src/components/entries/Tooltip.js

```diff
@@ -27,7 +26,7 @@ export default function TooltipWrapper(props) {
     return props.children;
   }
 
-  return <Tooltip { ...props } value={ value } forId={ prefixId(forId) } />;
+  return <Tooltip { ...props } value={ value } forId={ `bio-properties-panel-${ forId }` } />;
```

Comment:
This one is arguable. I'd rather keep the helper as it separates the prefixing from the rendering code.